libs/report/ingestion/src/lib/send-manifest.ts

```diff
@@ -0,0 +1,116 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client/core';
+import { readFileSync, statSync } from 'node:fs';
+import { TELEMETRY_INITIALIZE_MUTATION, TELEMETRY_REPORT_MUTATION } from './queries';
+import { type JSONValue } from './types';
+import { promptToProceedUploadFile } from './prompts';
+
+const client = new ApolloClient({
+  cache: new InMemoryCache(),
+  uri: 'https://api.nes.herodevs.com/graphql',
+});
+
+// I don't think this is the right key please change it :)
+const MANIFEST_TELEMETRY_KEY = 'd7:diagnostics:report';
```

Comment:
we'll def want to come up with a key here. 

`ingest:report` or `ingest:project:report` or something?

Reply:
sgtm but it needs to be configured in the backend

Reply:
PR to add the key https://github.com/neverendingsupport/monorepo/pull/359